api/preprints/serializers.py

```diff
@@ -574,3 +602,43 @@ def update(self, instance, validated_data):
     links = LinksField({
         'self': 'get_self_url',
     })
+
+
+class PreprintsInstitutionsRelationshipSerializer(BaseAPISerializer):
+    from api.institutions.serializers import InstitutionRelated  # Avoid circular import
+    data = ser.ListField(child=InstitutionRelated())
+
+    links = LinksField({
+        'self': 'get_self_url',
+        'html': 'get_related_url',
```

Comment:
I don't think this is right. I think it should just be:
```suggestion
        'self': 'get_related_url',
```

Because currently the output of the relationships/institutions endpoint is:
```
{
    "data": [],
    "links": {
        "self": "https://api.staging2.osf.io/v2/preprints/aqmer/",
        "html": "https://api.staging2.osf.io/v2/preprints/aqmer/institutions/"
    },
    "meta": {
        "version": "2.20"
    }
}
```

Reply:
I compared it with a project (see below). So I think `self` should point back to the relationship endpoint and HTML stays the same?

```json
"self": "https://api.staging2.osf.io/v2/preprints/aqmer/relationships/institutions/"
"html": "https://api.staging2.osf.io/v2/preprints/aqmer/institutions/"
```

For project:https://api.staging2.osf.io/v2/preprints/g5pdu/relationships/institutions/

```json
{
    "data": [
        {
            "type": "institutions",
            "id": "google"
        },
        {
            "type": "institutions",
            "id": "cos"
        }
    ],
    "links": {
        "self": "https://api.staging2.osf.io/v2/nodes/ngba5/relationships/institutions/",
        "html": "https://api.staging2.osf.io/v2/nodes/ngba5/institutions/"
    },
    "meta": {
        "version": "2.20"
    }
}
```

Reply:
Though this doesn't affect FE since it only uses this endpoint to `PUT`, let me take a look locally. I am also wondering why it's `html` instead of related, couldn't find it in `JSON:API` spec for links and maybe it should be `related`.